src/simsopt/util/permanent_magnet_helper_functions.py

```diff
@@ -351,7 +354,7 @@ def initialize_coils(config_flag, TEST_DIR, s, out_dir=''):
     return base_curves, curves, coils
 
 
-def calculate_on_axis_B(bs, s):
+def calculate_on_axis_B(bs, s, print_out=True):
     """
     Check the average, approximate, on-axis
     magnetic field strength to make sure the
```

Comment:
it's not really the magnetic axis that you're calculating the the modB average right?

Reply:
maybe `calculate_modB_on_major_radius` ?